src/components/domain/DrefShareModal/UserItem/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { DeleteBinFillIcon } from '@ifrc-go/icons';
+
+import Button from '#components/Button';
+import useTranslation from '#hooks/useTranslation';
+import { getUserName } from '#utils/domain/user';
+import { type User } from '#components/domain/UserSearchMultiSelectInput';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+interface Props {
+    className?: string;
+    userId: number;
+    onUserRemove: (item: number) => void;
+    userOptions: User[] | undefined | null;
```

Comment:
Lets not send the entire userOptions inside the userItem, but instead resolve and send it from outside.